app/src/main/java/app/nexd/android/ui/helper/requestOverview/HelperOverviewFragment.kt

```diff
@@ -56,7 +56,7 @@ class HelperOverviewFragment : Fragment() {
             HelpRequestBinder()
         )
 
-        viewModel.run {
+        vm.run {
```

Comment:
I think we should not scope this execution block to only the viewmodel, since it omits only 3 uses to access an observer in over 50 lines of code. Also the `vm` value is accessed in the inner block again. We could remove all use of `vm` in the inner block and use the more Kotlin style `with(vm){}`, but I think it helps readability for this whole block of code to name accessors explicitly like `vm.*`

Reply:
Absolutelly agree here. My change was just a rename here but i'll refactor that in this PR anyway.

Reply:
done